README.md

```diff
@@ -33,15 +45,37 @@ The unit tests make use of the mock server, so that a real server is not require
 
 ### Configuration
 
-**Server Setup**
+**Server Setup - V3 VisionMate Software**
+
 1. Enable the TCP/IP server: Open the VisionMate software and go to Export > Enable TCP/IP Server
 2. Display the TCP/IP tabs: Config > Show TCP/IP Tab
-3. Set a suffix character \(TCP/IP Server tab\). A suffix character marks the end of response data and is required for the client to function correctly. It is ideal to use a non-printable character to ensure that it is not found in barcodes or other data that the server may send. The suggested character is ASCII control character 3 \(the end of text character\), which can be chosen by entering '\[3\]'
+3. Set a suffix character \(TCP/IP Server tab\). A suffix character marks the end of response data and is required for
+   the client to function correctly. It is ideal to use a non-printable character to ensure that it is not found in
+   barcodes or other data that the server may send. The suggested character is ASCII control character 3 \(the end of
+   text character\), which can be chosen by entering '\[3\]'
+
+It is recommended to leave other settings at their default, but if you'd like to change anything else, ensure that the
+option is supported in the client (see below).
 
-It is recommended to leave other settings at their default, but if you'd like to change anything else, ensure that the option is supported in the client (see below).
+**Server Setup - V4 VisionMate Software**
+
+1. Enable the TCP/IP server: Export Options (yellow icon) > TCP Server \(a checkmark should appear to indicate it is
+   enabled\)
```

Comment:
Could we embed screenshots of these icons if they're unclear enough to need description? (Rack Formats too)